crates/pop-contracts/src/utils/contracts_node.rs

```diff
@@ -39,6 +34,51 @@ pub async fn is_chain_alive(url: url::Url) -> Result<bool, Error> {
 	}
 }
 
+/// Checks if the `substrate-contracts-node` binary exists
+/// or if the binary exists in pop's cache.
+/// returns:
+/// - Some("") if the standalone binary exists
+/// - Some(binary_cache_location) if the binary exists in pop's cache
+/// - None if the binary does not exist
+pub fn does_contracts_node_exist(cache: PathBuf) -> Option<PathBuf> {
+	let cached_location = cache.join(BIN_NAME);
+	if cmd(BIN_NAME, vec!["--version"]).run().map_or(false, |_| true) {
```

Comment:
I dont think we should output the version to the console, makes it messy imo.  Can redirect std_err as done elsewhere to silence this. Ideally the version should be read into a variable and returned via a tuple, so that it can potentially be used or output to the terminal in a controlled way.